pkg/metrics/metrics.go

```diff
@@ -62,6 +62,14 @@ var (
 			Help: "Number of times provider instance delete has failed.",
 		}, []string{"name", "namespace", "reason"},
 	)
+
+	// MachineHealthCheckNodesCovered is a Prometheus metric, which reports the number of nodes covered by MachineHealthChecks
+	MachineHealthCheckNodesCovered = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: "mapi_machinehealthcheck_nodes_covered",
+			Help: "Number of nodes covered by MachineHealthChecks",
+		}, []string{"name"},
+	)
```

Comment:
Could we put this into a separate var block as we have below for the Machine controller metrics? Would be better to have them separated into logic blocks per controller

Reply:
i had thought about that, i can definitely break it out.